app/javascript/groceries/components/check_in_items_list.vue

```diff
@@ -68,7 +69,8 @@ export default defineComponent({
 </script>
 
 <style lang="scss">
-.el-button.is-link {
+// double class in selector to increase specificity of this override
+.el-button.el-button.is-link {
```

Comment:
I noticed that, at least in my development environment, sometimes this style wasn't overriding the Element Plus CSS, as we intend for it to do.